pkgs/desktops/cinnamon/cinnamon-common/default.nix

```diff
@@ -153,6 +153,10 @@ stdenv.mkDerivation rec {
     sed "s|\"upload-system-info\"|\"${xapp}/bin/upload-system-info\"|g" -i ./files/usr/share/cinnamon/cinnamon-settings/modules/cs_info.py
 
     sed "s|/usr/bin/cinnamon-control-center|${cinnamon-control-center}/bin/cinnamon-control-center|g" -i ./files/usr/bin/cinnamon-settings
+
+    sed "s|/usr/bin/cinnamon-screensaver-command|/run/current-system/sw/bin/cinnamon-screensaver-command|g" \
```

Comment:
Shouldn't we use the package instead of `/run/current-system/sw`?

Reply:
Okay it looks like we cannot use the package, otherwise it will result in an infinite recursion. :disappointed: